### PR TITLE
Added lookcraft

### PR DESCRIPTION
**Pull request by zBenjx**
Added logo and metadata

### General:
* [x] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated metadata.json`)*
* [x] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [x] Folder with appropriate name has been created / updated (must not include any non-alpha characters or whitespace).
* [x] No changes were made to the file's formatting (4 spaces per tab).
* [x] There are no syntax errors.
* [x] There is no pre-existing mapping matching my id (check if there is an existing folder).
* [x] My field values match your requirements.
* You can view our patterns here: [metadata.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/metadata.schema.json), or take a look below and complete the field checklist:
  - [x] `id`: a lowercase string, which should match the folder name *(ex. `myserver`)*
  - [x] `name`: a string *(ex. `MyServerPvP`)*
  - [x] `addresses`: an array with lowercase strings *(ex. `["my.server", "your-server.com"]`)*
    - You do not need to specify subdomains, Lunar Client services automatically detect them.
  - [x] `primaryAddress`: an address *(ex. `my.server`)*
  - [x] `minecraftVersions`: an array with Minecraft versions as strings *(ex. `["1.18", "1.19"]` - Must be versions supported by Lunar Client)*
  - [x] `primaryMinecraftVersion`: a Minecraft version as a string *(ex. `1.18` - Must be versions supported by Lunar Client)*
  - [x] `primaryColor`: a hexademical color code that primarily distiguishes the server *(ex. `#00FFFF`)* 
  - [x] `secondaryColor`: a hexademical color code that accompanies the `primaryColor` of the server *(ex. `#FF0000`)*
  - [x] `primaryRegion`: the primary region where your server operates in *(ex. `NA`)*
  - [x] `regions`: a list of regions where you have servers located that service your players *(ex. `["NA", "EU", "AS"]`)*
  - [x] `gameTypes`: a list of games that describe the content on your server, must be a max of 3 listed *(ez. `["PVP", "UHC", "HCF"]`)*

#### Logo
* [x] My logo is a `png` file.
* [x] I have uploaded my logo to my server folder *(ex. `lunarnetwork`)* and named it `logo.png`.
* [x] My logo has a transparent background and is square (1:1 aspect ratio).
* [x] My logo is `512` pixels in width and height.
* [x] My logo is my own property and complies with relevant copyright/privacy laws.

#### Background
* [x] My background is a `png` file.
* [x] I have uploaded my background to my server folder *(ex. `lunarnetwork`)* and named it `background.png`.
* [x] My background has an aspect ratio of 16:9.
* [x] My background is atleast `1920` pixels in width and `1080` pixels in height (can be bigger, but must remain the same aspect ratio).
* [x] My background contains relevant content pertaining to the server, is my own property, and complies with relevant copyright/privacy laws.
* [x] My background doesn't contain any markings, text, or logos (unless part of a build).

### CODE REVIEW

**Review by wi — CHANGES_REQUESTED** [review]
You are missing a background.png file check the README for more info. Also under your addresses array, you don't need "play." in front of lookcraft.tk as Lunar Client will detect it.  Also are you only able to join your server on 1.8 and 1.19? You also need the following in your metadata.json:

- primaryColor
- secondaryColor
- gameTypes

**Comment by zBenjx** [comment]
Updated requirements.

**Review by wi — CHANGES_REQUESTED** [review]
Your background.png does not meet the minimum requirements of 1920px in width and 1080px in height (it's 1920x1027). Also as previously mentioned you're only able to join your server on ONLY 1.19 and 1.8? you need to add the other versions in between them if you can join with the different versions.

File: servers/lookcraft/metadata.json [hunk 1]
```diff
@@ -7,6 +7,16 @@
     "primaryAddress": "lookcraft.tk",
     "minecraftVersions": [
         "1.8",
+        "1.9",
+        "1.10",
```
Comment (wi) [inline servers/lookcraft/metadata.json:11]: Only versions supported by Lunar Client need to be here

**Review by wi — CHANGES_REQUESTED** [review]

File: servers/lookcraft/metadata.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+    "id": "lookcraft",
+    "name": "LookCraft",
+    "addresses": [
+        "lookcraft.tk"
+    ],
+    "primaryAddress": "lookcraft.tk",
+    "minecraftVersions": [
+        "1.8",
+        "1.12.2",
```
Comment (wi) [inline servers/lookcraft/metadata.json:10]:
```suggestion
1.12
```

Comment (wi) [inline servers/lookcraft/metadata.json:10]:
```suggestion
        "1.12",
````

File: servers/lookcraft/metadata.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        "1.16.5",
```
Comment (wi) [inline servers/lookcraft/metadata.json:11]:
```suggestion
        "1.16",
````

File: servers/lookcraft/metadata.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        "1.17.1",
```
Comment (wi) [inline servers/lookcraft/metadata.json:12]:
```suggestion
        "1.17",
````

File: servers/lookcraft/metadata.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        "1.18.1",
```
Comment (wi) [inline servers/lookcraft/metadata.json:13]:
```suggestion
        
````

File: servers/lookcraft/metadata.json [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+        "1.18.2",
+        "1.19",
+        "1.19.2"
```
Comment (wi) [inline servers/lookcraft/metadata.json:16]:
```suggestion
````

**Review by wi — DISMISSED** [review]

**Review by wi — APPROVED** [review]

**Pull request merged**